_episodes_rmd/05-day4.Rmd

```diff
@@ -1,135 +1,173 @@
 
-## Load vector files
-In this exercise, we have provided two vector data files: Wisconsin Annual Precipitation Average 1981-2010, and county outlines. You can work with these, or with your own vector datasets.
-```{r}
-wi_precip <- st_read("../data/precipitation/precip1981_2010_a_wi.shp")
-wi_county <- st_read("../data/WI_Counties2010/WI_Counties2010.shp")
-## County data is from here https://geo.btaa.org/catalog/4E677AF3-3FF2-43FA-8A58-3C345EC7F465
+## 1) Load Libraries 
+Load these libraries if you don't already have them (`dplyr`, `ggplot2`, `sf`, `rgdal`)
+
+
+> ## Solution
+> ```{r, eval=TRUE}
+> library(dplyr)
+> library(ggplot2)
+> library(sf)
+> library(rgdal)
+> ```
+{: .solution}
+
+## 2) Load vector files
+In this exercise, we have provided two capstone vector data files: Wisconsin Annual Precipitation Average 1981-2010, and county outlines. You can work with these, or with your own vector datasets.
+
+Filenames: precip1981_2010_a_wi.shp  and  WI_Counties2010.shp
+
+> ## Solution
+> ```{r, eval=TRUE}
+> wi_precip <- st_read("../data/precipitation/precip1981_2010_a_wi.shp")
+> wi_county <- st_read("../data/WI_Counties2010/WI_Counties2010.shp")
+> ## County data is from here  https://geo.btaa.org/catalog/4E677AF3-3FF2-43FA-8A58-3C345EC7F465
 ```
+{: .solution}
 
-## Explore metadata
-We have worked with commands to explore the geometry type, and projection of vector data files. Examples include:`st_geometry_type`, `st_crs`,  `st_bbox` 
-Explore the metadata of the shapefiles. We will work towards comnining these files in the same plot. Are these files suited to being plotted together? 
-```{r}
 
-st_geometry_type(wi_precip)
-st_crs(wi_precip)$proj4string
-st_bbox(wi_precip)
+## 3) Explore metadata
+We have worked with commands to explore the geometry type, and projection of vector data files. Examples include:`st_geometry_type()`, `st_crs()`,  `st_bbox()`, as well as reviewing the output produced in R when you imported the shapefiles. 
 
-st_geometry_type(wi_county)
-st_crs(wi_county)$proj4string
-st_bbox(wi_county)
+Explore the metadata of the shapefiles. We will work towards comnining these files in the same plot. Are these files suited to being plotted together - for example, are they in the same CRS and cover the same area? 
+
+> ## Solution
+> ```{r, eval=TRUE}
+> st_geometry_type(wi_precip)
+> st_crs(wi_precip)$proj4string
+> st_bbox(wi_precip)
+
+> st_geometry_type(wi_county)
+> st_crs(wi_county)$proj4string
+> st_bbox(wi_county)
 
 ```
+> The CRS for both vectors is the same. Both are in longlat projection, datum WGS84.
+> You may notice that the bounding box limits are slightly different. Since the limits are similar, we will continue working with them with the intent to plot them together, and make a note to inspect the plots and see why the limits are different. 
```

Comment:
I love this part! It's great to highlight that they should inspect their data and mentally flag a potential issue, but just because they aren't the same doesn't mean it won't work.